### PR TITLE
ocrvs-1904 Fixed add file feature for registrar

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

**Comment by sadmananik** [comment]
Peek 2021-12-09 21-00

**Comment by Zangetsu101** [comment]
The attachment option should be hidden for the registration clerk as per the requirements.

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx [hunk 1]
```diff
@@ -232,6 +245,9 @@ const mapDispatchToProps = (dispatch: Dispatch, props: IFullProps) => {
         variables.user.id = variables.user._fhirID
         delete variables.user._fhirID
       }
+      if (variables.user.signature) {
+        delete variables.user.signature.__typename
```
Comment (Zangetsu101) [inline packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:249]: This also should not be needed as I didn't find __typename inside signature.

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx (hunk 1 above)
Comment (sadmananik) [reply packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx]: Modified the code. Thanks for inform.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**